src/_live-rendering.ts

```
@@ -149,8 +149,8 @@ export function renderLiveDisplay(terminal: TerminalManager, block: SessionBlock
 
 	// Session details (indented)
 	const col1 = `${pc.gray('Started:')} ${startTime}`;
-	const col2 = `${pc.gray('Elapsed:')} ${prettyMs(elapsed * 60 * 1000, { compact: true })}`;
-	const col3 = `${pc.gray('Remaining:')} ${prettyMs(remaining * 60 * 1000, { compact: true })} (${endTime})`;
+	const col2 = `${pc.gray('Elapsed:')} ${Math.floor(elapsed / 60)}h ${Math.floor(elapsed % 60)}m`;
```

ESLint

[error] Unsafe call of a(n) `error` type typed value.

(ts/no-unsafe-call)